docs/api/covidcast-signals/google-symptoms.md

```diff
@@ -1,39 +1,39 @@
 ---
-title: Google Symptoms
+title: Google Search Trends symptoms dataset
 parent: Data Sources and Signals
 grand_parent: COVIDcast Epidata API
 ---
 
 # Google Symptoms
 {: .no_toc}
 
-* **Source name:** `google-symptoms`
+* **Source name:** `google-search-trends-symptoms-dataset`
```

Comment:
this is a technical identifier in the API, used in communication from clients to the server (as opposed to e.g. a descriptive label). changing this would break many things.

Reply:
So please feel free to keep it as is.

Reply:
I believe I submitted an updated request

Reply:
sounds good, thanks